spec/models/comment_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  let(:user) { User.create(name: 'Surafi', photo_url: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.') }
+  let(:post) { Post.create(author: user, title: 'The sky', text: 'Blue sky', likes_counter: 0) }
+
+  describe 'validations' do
+    it 'should validate presence of text' do
+      comment = Comment.create(post:, user:, text: 'Hello')
+      expect(comment).to be_valid
+    end
+
+    it 'is invalid if comments_counter is negative' do
+      post.comments_counter = -1
+      expect(post).to_not be_valid
+    end
+
+    it 'should be valid when comments_counter is zero' do
+      post.comments_counter = 0
+      expect(post).to be_valid
+    end
+
+    it 'should be valid when comments_counter is positive' do
+      post.comments_counter = 1
+      expect(post).to be_valid
+    end
+  end
+end
```

Comment:
- [ ] Great job adding the validations and testing them 👏. However, as stated in the requirement, all your models' methods should be tested too. So please, I would suggest you test the `update_post_comments_counter` method.

### Requirement :point_down: 
![Screenshot from 2023-12-15 16-53-41](https://github.com/Surafels/Blog-app/assets/89228743/2e79dda6-b073-458d-8fd9-45bc19a9024c)